website_odoo_debranding/static/description/index.html

```diff
@@ -0,0 +1,441 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
<?xml version="1.0" encoding="utf-8"?>
It gives an error, there is no need

Reply:
<?xml version="1.0" encoding="utf-8"?> is added in the index file by the precommit run, that i perform before commiting the migration part. Should i delete this manually? or maybe an issue for the pre commit setup?

Reply:
@pedrobaeza 
What do you think concerning the mentioned line and how to get rid of it. Manually or adjusting pre commit? Or do I miss something?

Reply:
It's just a warning, so let's continue with it for now. The README generator should be changed to not generate this line, but that's not done yet. If someone wants to contribute with that improvement (only for 17.0).

Reply:
Alright.. thank for having a look and the feedback.